src/java.desktop/macosx/classes/com/apple/laf/AquaComboBoxButton.java

```diff
@@ -33,6 +33,8 @@
 import apple.laf.JRSUIState;
 import apple.laf.JRSUIConstants.*;
```

Comment:
I think we should expand all wild card imports.

Reply:
Updated.